app/views/pieces/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>move your piece</h1>
+
+<table id="chessboard-2" class="chessboard">
+  <!-- 1, 8 = top left -->
+  <% (1..8).to_a.reverse.each do |i| %>
+    <tr>
+      <% (1..8).each do |j| %>
+        <td data-y-pos="" data-x-pos="<%= j %>">
```

Comment:
I think we need to add <%= i %> into data-y-pos="" like I have below:

<td data-y-pos="<%= i %>" data-x-pos="<%= j %>">